lib/exercises.rb

```diff
@@ -1,19 +1,47 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 The time complexity is O(n) if the strings are limited in size, if they're not because you're sorting the time complexity is O(m * n log n), where m is the number of strings and n is the length of the strings.